pyntc/devices/base_device.py

```diff
@@ -9,6 +9,11 @@
 
 
 def fix_docs(cls):
+    """Create docstring at runtime.
+
+    Returns:
+        class: Returns the class passed in.
+    """
```

Comment:
I discussed the appropriate way to document returning the class that was passed with Dr. X. We weren't too sure the proper verbiage.  As this function will be going away after all docsrings are implemented I just put a very basic docstring.

Reply:
I still have not found anything relevant.